R/util.R

```diff
@@ -785,9 +790,19 @@ pivot_ <- function(df, formula, value_col = NULL, fun.aggregate = mean, fill = N
     dplyr::do_(.dots=setNames(list(~pivot_each(.)), tmp_col)) %>%
     unnest_with_drop_(tmp_col)
 
-  # replace NA values in missing columns in some groups with fill
+  if(length(rows) == 1){
```

Comment:
When rows are more than 2, it's concatenated, so the data type can't be converted

![image](https://cloud.githubusercontent.com/assets/6638312/26575178/16976668-455f-11e7-94a1-23eb1f31a916.png)